data/abilities/collection/T1114__email-collection.yml

```diff
@@ -0,0 +1,13 @@
+- id: 5e1f26ab-ca46-4db5-a389-3647cebb7717
+  name: Email Collection
+  description: Ability for Email Collection (ATT&CK T1114). Generated from Atomic
+    Red Team procedures; review and test safely before use.
+  tactic: collection
+  technique:
+    attack_id: T1114
+    name: Email Collection
+  platforms:
+    windows:
+      psh:
+        command: 'powershell -executionpolicy bypass -command "#{file_path}\Get-Inbox.ps1"
```

Comment:
if this is referencing a payload it needs to also be in the PR and marked as such in the yaml

If used as a payload, the `file_path` fact is no longer relevant since the agents will typically download the payloads to the current working directory